test/src/main/java/com/demoqa/MainPage.java

```diff
@@ -0,0 +1,33 @@
+package com.demoqa;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+public class MainPage extends BasePage {
+
+    private static final By ELEMENTS_CARD = By.xpath("//div[@class='card-body']/h5[text()='Elements']");
+    private static final String WIDGETS_CARD = "//div[@class='card-body']/h5[text()='Widgets']";
```

Comment:
And by the way, in case of `String` type this is not WIDGETS_CARD, this is WIDGETS_CARD_XPATH, the name of the constant should represent it's meaning